### DISCUSSION TITLE
Lesson 4: Error Received On Visual Studio Code

### DISCUSSION THREAD

**Discussion by elliottmcknight** [smartcontractkit/full-blockchain-solidity-course-py#1684] [Q&A] [answered]
When attempting to write anything in Visual Studio Code, I am receiving this error message. I am currently using a Mac on the latest version of Visual Studio Code. Has anyone run into this problem? If so, could you help me resolve the issue?

```
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

**Comment by cromewar** [comment, marked as answer]
Hello @elliottmcknight 
Seems to be a problem with your editor, most probably a extension not written well I recommend you the following.
1. Open Settings.json and delete all it's content.
2. Reinstall VScode.

**Reply by elliottmcknight** [reply]
I got it figured out! Thank you!
